exchanges/subscription/subscription_test.go

```diff
@@ -134,3 +134,12 @@ func TestSetPairs(t *testing.T) {
 	s.SetPairs(currency.Pairs{btcusdtPair})
 	assert.Equal(t, "BTCUSDT", s.Pairs.Join(), "SetPairs should set a key correctly")
 }
+
+// TestAddPairs exercises AddPairs
+func TestAddPairs(t *testing.T) {
+	s := &Subscription{}
```

Comment:
Looks like these could also all be parallel

Reply:
Fixed b6081c7

Sorry about that.